### PR TITLE
add: `g4music`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

**Comment by soumyatheman** [comment]
@Henryws can you check please where is the issue?

File: packages/g4music/g4music.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+build_depends="meson valac"
+depends="libgtk-4-dev libgstreamer1.0-dev libadwaita-1-dev"
+url="https://gitlab.gnome.org/neithern/g4music/-/archive/v${version}/g4music-v${version}.tar.gz"
+repology=("project: g4music")
+description="Play your music in an elegant way"
+hash=""
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
+prepare() {
+  true
+}
```
Comment (Henryws) [inline packages/g4music/g4music.pacscript:13]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
> @Henryws can you check please where is the issue?

Take a look at the CIs.

**Comment by soumyatheman** [comment]
This package only works in Ubuntu 22.10, or distros with libadwaita. And everywhere it clearly says, it failed download the packages.

File: packages/g4music/g4music.pacscript [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name="g4music"
+gives="g4music"
+version="1.8.1"
+build_depends="meson valac libgtk-4-dev libgstreamer1.0-dev libadwaita-1-dev"
+depends="libgtk-4-dev libgstreamer1.0-dev libadwaita-1-dev"
+url="https://gitlab.gnome.org/neithern/g4music/-/archive/v${version}/g4music-v${version}.tar.gz"
+repology=("project: g4music")
+description="Play your music in an elegant way"
+hash=""
```
Comment (Henryws) [inline packages/g4music/g4music.pacscript:9]: Add a hash.

**Pull request closed without merging**